apollo-router/src/query_planner/execution.rs

```diff
@@ -42,7 +42,7 @@ impl QueryPlan {
         context: &'a Context,
         service_factory: &'a Arc<SubgraphServiceFactory>,
         supergraph_request: &'a Arc<http::Request<Request>>,
-        schema: &'a Schema,
+        schema: &'a Arc<Schema>,
```

Comment:
Why this change?

Reply:
This enables removing `#[derive(Clone)]` on `Schema`, which in turn is not really necessary for this iteration of this PR since we don’t keep `ApolloCompiler` inside `Schema`. But IMO it’s still better to avoid cloning multiple `HashMap`s and instead clone an `Arc`.

Reply:
oh, right, we only needed a reference before, but with `@defer` the schema is cloned